PicoVR Unity SDK/chapter_seven.txt

```diff
@@ -1,281 +1,602 @@
+2.	Apply to be an *individual* or *enterprise* developer. Once your submission is reviewed, we’ll provide feedback within three working days. Please check on the status of your submission on the developer platform within this time frame.
 
-7.2 	Open Screen Fade of the Scenes
------------------------------------------------
+3.	Check the merchant ID. After becoming a developer, click on the nickname in the upper right corner to view the developer ID, which will be your unique merchant ID for the payment system:
 
-The SDK provides Open Screen Fade of the Scenes to be turned on through options as follows
+.. image:: _static/8.1.png
 
-.. image:: _static/7.2.png
+Figure 7.1 Checking the Merchant ID
 
-Figure 7.2 Opening of Screen Fade
+4.	Obtain the corresponding strings.
 
-Figure 7.3 can be used to set the Fade Time and Fade Color of the Scenes (time unit is S)
+5.	Enter the application creation step from the administration screen. Click the button to create an application and select a platform to publish to:
 
-.. image:: _static/7.3.png
+.. image:: _static/7.1a.png
 
-Figure 7.3 Fade Time and Fade Color after Opening the Scenes
+Figure 7.1 Selecting a Platform to Publish the App to.
 
-7.3 	Limited frame rate
--------------------------------------
+6.	Complete the app-related information for the corresponding platform:
 
-The SDK provides the function of limiting frame rate, which is opened by unchecking the Use Default FPS option as follows
+.. image:: _static/8.2.png
 
-.. image:: _static/7.4.png
+Figure 7.2 Entering the App-related Information
 
-Figure 7.4 Opening of limited frame rate
+Enter the values for each field carefully because the app type cannot be modified once completed. If there exists an inside-props payment in your application, you must use the developer backend to add a merchandise code for unified management.
 
-Then enter the desired frame rate value in the FPS text box.
+7.1.1 	Description of Payments
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-7.4 	Configure the size of EyeBuffer
----------------------------------------------
+Pico currently has two types of payment methods: *P currency payment (application type)*, and *commodity code payment (game type)*. An application can only use one of the two payment methods.
+After the app has been created successfully, the developer platform will assign it a number of strings including an ``APPID``, ``APP KEY``, ``APP Secret``, and a developer ID:
 
-The SDK provides the function of configuring the size of the Eyebuffer, which is opened by unchecking the Use Default RenderTexture option as follows
+.. image:: _static/8.3.png
 
-.. image:: _static/7.5.png
+Figure 7.3 APP ID, APP KEY, APP Secret
 
-Figure 7.5 Configuration size of Eyebuffer
+Once these strings have been provided, select In-Game Payment Settings to configure in-game purchase information:
 
-7.5 	Custom Application Launch Animation
----------------------------------------------
+.. image:: _static/7.3a.png
```

Comment:
Remove this line because in-game purchase is not provided in [us-dev.picovr.com](url)
Starting from here...